src/base1/test-dbus-common.js

```diff
@@ -1130,4 +1130,28 @@ function dbus_track_tests(channel_options, bus_name) {
                     });
             });
     });
+
+    QUnit.asyncTest("receive fd", function() {
```

Comment:
This tests receiving a read fd. It would be good to also have a test for getting a write fd, unless this shouldn't be supported (for now) -- but then that should be documented.